Hymns/SearchView.swift

```diff
@@ -0,0 +1,30 @@
+import SwiftUI
+
+struct SearchView: View {
+    var hymns = hymnTestData
+    @State private var searchText: String = ""
+
+    //Currently we have to use UIKIT to change the unselected tabicons from grey to black.
+    init() {
+        UITabBar.appearance().unselectedItemTintColor = UIColor.black
```

Comment:
I am a bit confused why UITabBar's appearance lives in SearchView. Shouldn't it be in ContentView? That seems to be the container of all the tabs right, while this is just the search view?

Reply:
Yeah, good question. This has been driving me absolutely crazy. I have spent hours trying to figure out why this will not work on our Home Screen view... I'm still not sure how to get it working on the main screen view at this point. I went ahead and moved it to HomeView which makes at least a little more sense then being in searchview.